tasks/kernels/run.py

```diff
@@ -0,0 +1,230 @@
+import math
```

Comment:
This is an amalgamation of all the files in the `run` directory of the `experiment-kernels` repo: https://github.com/faasm/experiment-kernels/tree/master/run

The scripts in there, especially `all_faasm.py` and `all_native.py` contained a lot of duplication, and this should be all that's needed.